src/components/DeleteAccountModal/DeleteAccountModal.tsx

```diff
@@ -56,7 +56,6 @@ const DeleteAccountModal = () => {
     if (!location.search.includes('delete-account-confirmation') && enteredPassword) {
       // handle back button
       setEnteredPassword('');
-      deleteAccount.reset();
```

Comment:
why is this removed?

Reply:
![Screenshot 2023-10-31 at 14 53 02](https://github.com/jwplayer/ott-web-app/assets/104987342/03c221ba-a35c-4df0-b3e7-2f6fa728007f)

Reply:
Because i found bag, that when you go cancel on the modal it gets into infinity loop.